app/assets/javascripts/student_profile/pdf/student_profile_pdf.js

```diff
@@ -1,74 +1,10 @@
 import _ from 'lodash';
-import Highcharts from 'highcharts';
 import * as GraphHelpers from '../../helpers/graph_helpers';
+import generateReportGraph from './GenerateReportGraph';
 
 (function() {
   window.shared || (window.shared = {});
 
-  function generateGraph(containerSelector, yAxisLabel, xAxisSettings, title, dataSeries){
-
-    const stacking = (dataSeries.length > 1)
-      ? "normal"
-      : "";
-
-    $(containerSelector).highcharts({
-      chart: {
-        type: 'column'
-      },
-      title: {
-        text: title
-      },
-      credits: false,
-      exporting: {
-        enabled: false
-      },
-      plotOptions: {
-        // This sets charts to render immediately, so we can
-        // synchronously set window.status and pass control back
-        // to wkhtmltopdf to render the PDF.
-        series: { animation: false },
-        column: {
-          stacking: stacking,
-          dataLabels: {
-            enabled: true,
-            color: (Highcharts.theme && Highcharts.theme.dataLabelsColor) || 'white',
-            formatter: function(){
-              const val = this.y;
```

Comment:
@alexsoble I refactored it from here so I thought `this.y` is correct. Is there something I am missing? 🤔